wls-gui-wahllokalsystem/tests/composables/wahlvorstand/wahlvorstandservice.spec.ts

```diff
@@ -88,7 +90,9 @@ describe("WahlvorstandService.ts", () => {
       mockDefinitions.getWahlvorstand.mockReturnValue(
         Promise.resolve({ data: createWahlvorstandDTO() })
       );
-      const result = await unitUnderTest.getWahlvorstand("wahlbezirkID", true);
+      const result = await unitUnderTest.getWahlvorstand("wahlbezirkID", {
+        sendNotification: true,
+      });
 
       expect(result).toStrictEqual(mockedMappedWahlvorstand);
       expect(mockDefinitions.addNotification.mock.calls).toEqual([
```

Comment:
Den String `wahlbezirkID` könnte man noch in eine lokale variable auslagern

```suggestion
      expect(mockDefinitions.getWahlvorstand.mock.calls).toStrictEqual([
        ["wahlbezirkID", false],
      ]);
      expect(mockDefinitions.addNotification.mock.calls).toEqual([
```

Reply:
[88c9b77](https://github.com/it-at-m/Wahllokalsystem/pull/1399/commits/88c9b77e37fe052ec36e81054d605343d9aa0c18)